### PR TITLE
WORKDIR set in Dockerfile and volmes for backup reasons

### PR DESCRIPTION
**Pull request by Shikachuu**
Hey guys,
For backup and other reasons I think it is a must to mount the `pages` and `data` directory, I corrected the README.md to make it clear for everyone.

Also for relative path usage for situations like this I also make a change in the Dockerfile  and set the `WORKDIR` to `/var/www/app/`

### CODE REVIEW

**Review by creecros — APPROVED** [review]
This is fine to add

**Review by creecros — CHANGES_REQUESTED** [review]
I'd rather add the usage example instead of modifying the basic. Binding folders to the host is not actually necessary. I see the convenience, but know that the added bind commands would fail on my own system, therefore, it's not the basic usage.

File: README.md [hunk 1]
```diff
@@ -44,11 +44,12 @@ composer create-project kenlog/docpht
...
 ```
 docker pull docpht/docpht:v1.0.0
-docker run -d --name docpht -p 80:80 -t docpht/docpht:v1.0.0
+docker run -d --name docpht -p 80:80 -t docpht/docpht:v1.0.0 --volume ./data:/var/www/app/data ./pages:/var/www/app/pages
 ```
 
 ## Wrapping Up
-That is all you need to get started. Boom! 
+That is all you need to get started. Boom!
+The command belowe just spins up a container for you, with backup folders from `data/` and `pages/` which contains your saved notes.
```
Comment (creecros) [inline README.md]: Would it not be "above" instead of "belowe", also "belowe" should be spelled "below"

**Comment by Shikachuu** [comment]
> 
> 
> I'd rather add the usage example instead of modifying the basic. Binding folders to the host is not actually necessary. I see the convenience, but know that the added bind commands would fail on my own system, therefore, it's not the basic usage.

Done, thanks for the response :)

**Comment by kenlog** [comment]
> No, still not what I meant.
> 
> i.e.
> Basic usage:
> Yadda yadda yadda
> 
> Example binding volumes:
> Your example...
> 
> Need to "add" your example, not modify the basic.
> 
> Make sense?

@Shikachuu

**Pull request closed without merging**